wordpress-mobile/WordPress-Android
# Prepublishing Nudges : Toolbar Improvements

Fixes #11698

## Solution

1. Increased the touch target of the back and close button that is within the `prepublishing toolbar` by wrapping them in a layout with the default touch target size. 

2. Since the clicks are now being done on the layout I had to replace the `ImageView` references with `View` and in other instances, I simply utilized the auto-generated ktx view references.  

3. Fixed the constraints of the buttons and the fixed the title positioning. 

## Testing
1. Create a new post. 
2. Click publish.
3. Click on both Tags & Publish Date. 
4. Ensure the back and close buttons for both have a good touch area; they should be easy to click. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/81122617-dd0df580-8ef6-11ea-8451-bce34a8363b7.png" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
